gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java

```diff
@@ -188,9 +188,10 @@ void updateConstraintStatus(
   @GetMapping(path = "/application/{application}")
   Map getApplicationDetails(
       @PathVariable("application") String application,
-      @RequestParam(value = "includeDetails", required = false, defaultValue = "false")
-          Boolean includeDetails) {
-    return keelService.getApplicationDetails(application, includeDetails);
+      @RequestParam(name = "includeDetails", required = false, defaultValue = "false")
+          Boolean includeDetails,
+      @RequestParam(name = "entities", required = false, defaultValue = "") List<String> entities) {
```

Comment:
can we make the default value `resources` here? as of right now if we ship this and then the new keel version, I think deck is going to stop getting back resource summaries. Just checked the new keel we've got up and running internally and if i hit it without an `entities` list I get no summaries at all.

Reply:
OK.